groundX-doc-pipeline/run_evaluation_cli.py

```
@@ -0,0 +1,234 @@
+import argparse
+import mimetypes
+from pathlib import Path
+from typing import List, Dict, Any
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)